<!DOCTYPE html>





<html lang="en">
<head>
  <meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=2">
<meta name="theme-color" content="#222">
<meta name="generator" content="Hexo 3.9.0">
  <link rel="apple-touch-icon" sizes="180x180" href="/images/apple-touch-icon-next.png?v=7.4.1">
  <link rel="icon" type="image/png" sizes="32x32" href="/images/favicon-32x32-next.png?v=7.4.1">
  <link rel="icon" type="image/png" sizes="16x16" href="/images/favicon-16x16-next.png?v=7.4.1">
  <link rel="mask-icon" href="/images/logo.svg?v=7.4.1" color="#222">

<link rel="stylesheet" href="/css/main.css?v=7.4.1">


<link rel="stylesheet" href="/lib/font-awesome/css/font-awesome.min.css?v=4.7.0">


<script id="hexo-configurations">
  var NexT = window.NexT || {};
  var CONFIG = {
    root: '/',
    scheme: 'Mist',
    version: '7.4.1',
    exturl: false,
    sidebar: {"position":"left","display":"post","offset":12,"onmobile":false},
    copycode: {"enable":false,"show_result":false,"style":null},
    back2top: {"enable":true,"sidebar":false,"scrollpercent":false},
    bookmark: {"enable":false,"color":"#222","save":"auto"},
    fancybox: false,
    mediumzoom: false,
    lazyload: false,
    pangu: false,
    algolia: {
      appID: '',
      apiKey: '',
      indexName: '',
      hits: {"per_page":10},
      labels: {"input_placeholder":"Search for Posts","hits_empty":"We didn't find any results for the search: ${query}","hits_stats":"${hits} results found in ${time} ms"}
    },
    localsearch: {"enable":false,"trigger":"auto","top_n_per_article":1,"unescape":false,"preload":false},
    path: '',
    motion: {"enable":true,"async":false,"transition":{"post_block":"fadeIn","post_header":"slideDownIn","post_body":"slideDownIn","coll_header":"slideLeftIn","sidebar":"slideUpIn"}},
    translation: {
      copy_button: 'Copy',
      copy_success: 'Copied',
      copy_failure: 'Copy failed'
    },
    sidebarPadding: 40
  };
</script>

  <meta name="description" content="Fast p-value calculation for HC and BJ tests In genome-wide association studies (GWASs), scientists commonly group multiple SNVs together based on their chromosomal site, then test the association bet">
<meta property="og:type" content="website">
<meta property="og:title" content="Research">
<meta property="og:url" content="http://yoursite.com/research/index.html">
<meta property="og:site_name" content="Wenjia Wang">
<meta property="og:description" content="Fast p-value calculation for HC and BJ tests In genome-wide association studies (GWASs), scientists commonly group multiple SNVs together based on their chromosomal site, then test the association bet">
<meta property="og:locale" content="en">
<meta property="og:image" content="http://yoursite.com/research/Prediction.anchor.pdf">
<meta property="og:updated_time" content="2022-01-06T19:42:36.364Z">
<meta name="twitter:card" content="summary">
<meta name="twitter:title" content="Research">
<meta name="twitter:description" content="Fast p-value calculation for HC and BJ tests In genome-wide association studies (GWASs), scientists commonly group multiple SNVs together based on their chromosomal site, then test the association bet">
<meta name="twitter:image" content="http://yoursite.com/research/Prediction.anchor.pdf">
  <link rel="canonical" href="http://yoursite.com/research/">


<script id="page-configurations">
  // https://hexo.io/docs/variables.html
  CONFIG.page = {
    sidebar: "",
    isHome: false,
    isPost: false,
    isPage: true,
    isArchive: false
  };
</script>

  <title>
      Research | Wenjia Wang
    
  </title>
  








  <noscript>
  <style>
  .use-motion .brand,
  .use-motion .menu-item,
  .sidebar-inner,
  .use-motion .post-block,
  .use-motion .pagination,
  .use-motion .comments,
  .use-motion .post-header,
  .use-motion .post-body,
  .use-motion .collection-header { opacity: initial; }

  .use-motion .site-title,
  .use-motion .site-subtitle {
    opacity: initial;
    top: initial;
  }

  .use-motion .logo-line-before i { left: initial; }
  .use-motion .logo-line-after i { right: initial; }
  </style>
</noscript>

</head>

<body itemscope itemtype="http://schema.org/WebPage">
  <div class="container use-motion">
    <div class="headband"></div>

    <header class="header" itemscope itemtype="http://schema.org/WPHeader">
      <div class="header-inner"><div class="site-brand-container">
  <div class="site-meta">

    <div>
      <a href="/" class="brand" rel="start">
        <span class="logo-line-before"><i></i></span>
        <span class="site-title">Wenjia Wang</span>
        <span class="logo-line-after"><i></i></span>
      </a>
    </div>
        <p class="site-subtitle">xxvwang@ucdavis</p>
      
  </div>

  <div class="site-nav-toggle">
    <div class="toggle" aria-label="Toggle navigation bar">
      <span class="toggle-line toggle-line-first"></span>
      <span class="toggle-line toggle-line-middle"></span>
      <span class="toggle-line toggle-line-last"></span>
    </div>
  </div>
</div>


<nav class="site-nav">
  
  <ul id="menu" class="menu">
      
      
        
        
        <li class="menu-item menu-item-home">
      
    

    <a href="/" rel="section"><i class="fa fa-fw fa-home"></i>Home</a>

  </li>
      
    
      
      
        
        
        <li class="menu-item menu-item-research">
      
    

    <a href="/research/" rel="section"><i class="fa fa-fw fa-heartbeat"></i>research</a>

  </li>
      
    
      
      
        
        
        <li class="menu-item menu-item-projects">
      
    

    <a href="/projects/" rel="section"><i class="fa fa-fw fa-tags"></i>projects</a>

  </li>
      
    
  </ul>

    

  
    
    
  
    
    
  
    
    
  
  

</nav>
</div>
    </header>

    
  <div class="back-to-top">
    <i class="fa fa-arrow-up"></i>
    <span>0%</span>
  </div>


    <main class="main">
      <div class="main-inner">
        <div class="content-wrap">
          <div class="content">
            

  <div class="posts-expand">
    
    
    
    <div class="post-block" lang="en">
      <header class="post-header">

<h1 class="post-title" itemprop="name headline">Research
  

</h1>

<div class="post-meta">
  

</div>

</header>

      
      
      
      <div class="post-body">
          <h2 id="Fast-p-value-calculation-for-HC-and-BJ-tests"><a href="#Fast-p-value-calculation-for-HC-and-BJ-tests" class="headerlink" title="Fast p-value calculation for HC and BJ tests"></a>Fast p-value calculation for HC and BJ tests</h2><p><font size="2"> In genome-wide association studies (GWASs), scientists commonly group multiple SNVs together based on their chromosomal site, then test the association between each SNV-set and phenotype one at a time. This results in tens of thousands of set tests and consequently more stringent type I error control for each test. Besides, only a small proportion of SNVs is expected to be phenotype-associated in each SNV-set. Solving this problem requires not only a powerful multiple hypothesis test for the sparse signal detection, but also a fast and accurate p-value or threshold computation method at stringent significance level.</font></p>
<p><font size="2"> Higher Criticism (HC) and Berk-Jones (BJ) are two typical multiple hypothesis test statistics, and generally used to detect weak and sparse signals since they were proved asymptotically powerful. However, there is no simple formula of the null distribution, and current literature has limitations in computing the null distribution, such as numerical instability, inflexibility and failure to recover the whole distribution. The contributions of the research include proposing the cross-entropy based importance sampling with appropriate importance density to recover the tail distribution for HC and BJ of various supremum domains, modifying the analytic methods to overcome the numerical instability and inflexibility w.r.t. supremum domains, and finally constructing the quantile-probability libraries and smooth curves to reveal the whole null distribution for HC and BJ of different supremum domains over a wide range of dimensions so that it can be much faster for vectorizing computation of p-values given a huge number of observed statistics, or finding the thresholds given a family-wise significance level. <a href="https://wenjiaking.github.io/p_valueComputationENAR.pdf" target="_blank" rel="noopener">[Paper Manuscript]</a></font></p>
<h2 id="Pipeline-for-long-read-sequencing-data"><a href="#Pipeline-for-long-read-sequencing-data" class="headerlink" title="Pipeline for long-read sequencing data"></a>Pipeline for long-read sequencing data</h2><p><font size="2"> The pipeline is developed based on R and some existing tools such as the aligner MINIMAP2. Though immature it is, the pipeline can identify whether the long read sequence is a known/normal isoform and whether it is a fusion or not. In addition, it provides other information including the chromosome position, strand, gene, and fusion point if it is a fusion, etc. </font></p>
<p><font size="2"> Currently, the pipeline works well in our simulation as the figures show. The data is simulated by PBSIM tool, including six types of novel transcript and the normal transcript from 1000 randomly selected genes, followed by mixing each novel type of reads with normal reads to obtain six data sets. Apply the developed pipeline and evaluate its ability of detecting novel reads based on the ground truth. In addition, get some insights in the sensitivity to anchor length by tolerant some error bps. </font><br><img src="/research/Prediction.anchor.pdf" alt="alt"></p>
<p><font size="2"> Next stage is to evaluate the pipeline on fusion detection, analyze its sensitivity to anchor length, improve it by combining multiple aligners (e.g. STARLong), finalize the pipeline and apply it to our real long sequence cancer data. </font></p>
<h2 id="Neurons-extraction-from-in-vivo-calcium-imaging-data"><a href="#Neurons-extraction-from-in-vivo-calcium-imaging-data" class="headerlink" title="Neurons extraction from in-vivo calcium imaging data"></a>Neurons extraction from in-vivo calcium imaging data</h2><p><font size="2"> Calcium imaging is an important technique for monitoring the activity of a large number of neurons simultaneously. Repeatedly scanning a single image plane yields a time series of 2D images. In fact, the characteristics of the films are different across neuronal compartments recorded (eg. cell body, axonal boutons, dendritic compartments, and spines), field of view (FOV), spatial and temporal resolution. In addition, any method of analyzing calcium imaging data must be tailored to, not only fitting the features of images, but also meeting the demands of the specific scientific application. </font></p>
<p><font size="2"> Though a variety of unsupervised learning basis methods have been proposed, the difficulties in efficiently handling out-of-focus or neuropil contamination due to low z-resolution still exist. Besides, as the studying synaptic transmission and dendritic integration in vivo springs up, statisticians are expected to propose efficient approaches to extract the activities of dendritic spines or even axonal boutons. However, only few papers focus on these issues now. </font></p>
<p><font size="2"> The research purpose is to develop approaches to simultaneously identify neurons and estimate calcium concentrations from in-vivo calcium imaging data corrupted by complex background based on the existing methods, and try to make some improvements in dealing dendrites imaging data. (Details: <a href="https://wenjiaking.github.io/Review.pdf" target="_blank" rel="noopener">[Review Manuscript]</a> and <a href="https://github.com/wenjiaking/SCALPEL_Simulation" target="_blank" rel="noopener">[Code]</a>)</font></p>

        
      </div>
      
      
      
    </div>
    

    
    
    
  </div>


          </div>
          

        </div>
          
  
  <div class="toggle sidebar-toggle">
    <span class="toggle-line toggle-line-first"></span>
    <span class="toggle-line toggle-line-middle"></span>
    <span class="toggle-line toggle-line-last"></span>
  </div>

  <aside class="sidebar">
    <div class="sidebar-inner">
        
        
        
        
      

      <ul class="sidebar-nav motion-element">
        <li class="sidebar-nav-toc">
          Table of Contents
        </li>
        <li class="sidebar-nav-overview">
          Overview
        </li>
      </ul>

      <!--noindex-->
      <div class="post-toc-wrap sidebar-panel">
          <div class="post-toc motion-element"><ol class="nav"><li class="nav-item nav-level-2"><a class="nav-link" href="#Fast-p-value-calculation-for-HC-and-BJ-tests"><span class="nav-number">1.</span> <span class="nav-text">Fast p-value calculation for HC and BJ tests</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Pipeline-for-long-read-sequencing-data"><span class="nav-number">2.</span> <span class="nav-text">Pipeline for long-read sequencing data</span></a></li><li class="nav-item nav-level-2"><a class="nav-link" href="#Neurons-extraction-from-in-vivo-calcium-imaging-data"><span class="nav-number">3.</span> <span class="nav-text">Neurons extraction from in-vivo calcium imaging data</span></a></li></ol></div>
        
      </div>
      <!--/noindex-->

      <div class="site-overview-wrap sidebar-panel">
        <div class="site-author motion-element" itemprop="author" itemscope itemtype="http://schema.org/Person">
  <p class="site-author-name" itemprop="name">Wenjia Wang</p>
  <div class="site-description" itemprop="description"></div>
</div>
<div class="site-state-wrap motion-element">
  <nav class="site-state">
      <div class="site-state-item site-state-posts">
        
          <a href="/archives">
        
          <span class="site-state-item-count">1</span>
          <span class="site-state-item-name">posts</span>
        </a>
      </div>
    
  </nav>
</div>



      </div>

    </div>
  </aside>
  <div id="sidebar-dimmer"></div>


      </div>
    </main>

    <footer class="footer">
      <div class="footer-inner">
        

<div class="copyright">
  &copy; 
  <span itemprop="copyrightYear">2022</span>
  <span class="with-love">
    <i class="fa fa-user"></i>
  </span>
  <span class="author" itemprop="copyrightHolder">Wenjia Wang</span>
</div>
  <div class="powered-by">Powered by <a href="https://hexo.io" class="theme-link" rel="noopener" target="_blank">Hexo</a> v3.9.0
  </div>
  <span class="post-meta-divider">|</span>
  <div class="theme-info">
      
    Theme – <a href="https://mist.theme-next.org" class="theme-link" rel="noopener" target="_blank">NexT.Mist</a> v7.4.1
  </div>

        












        
      </div>
    </footer>
  </div>

  
  <script src="/lib/anime.min.js?v=3.1.0"></script>
  <script src="/lib/velocity/velocity.min.js?v=1.2.1"></script>
  <script src="/lib/velocity/velocity.ui.min.js?v=1.2.1"></script>
<script src="/js/utils.js?v=7.4.1"></script><script src="/js/motion.js?v=7.4.1"></script>
<script src="/js/schemes/muse.js?v=7.4.1"></script>

<script src="/js/next-boot.js?v=7.4.1"></script>



  





















  

  

  

</body>
</html>
